packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -99,28 +125,28 @@ export const Dropdown = memo(
       strategy: positionFixed ? 'fixed' : 'absolute',
     });
 
-    const renderToggle = useMemo(() => {
-      return (
-        isValidElement(toggle) &&
-        cloneElement(toggle, {
-          ...getToggleButtonProps({
-            'aria-expanded': isOpen, // Because of memoization, we need to manually set this option
-            disabled,
-            ref: referenceRef,
-          }),
-        })
-      );
-    }, [disabled, getToggleButtonProps, isOpen, toggle]);
+    const clonedToggle =
+      isValidElement(toggle) &&
+      cloneElement(toggle, {
+        ...getToggleButtonProps({
+          'aria-haspopup': 'menu',
+          // Downshift sets this to a label id that doesn't exist
+          'aria-labelledby': undefined,
+          disabled,
+          ref: referenceRef,
+          role: 'button',
+        }),
+      });
 
     return (
       <StyledBox>
-        {renderToggle}
+        {clonedToggle}
         <Box ref={popperRef} style={styles.popper} {...attributes.poppper} zIndex="popover">
           <List
             {...props}
             autoWidth={autoWidth}
             getItemProps={getItemProps}
-            getMenuProps={getMenuProps}
+            getMenuProps={() => getMenuProps({ role: 'menu' })}
```

Comment:
We are retrofitting downshift to be a dropdown menu, so had to apply the appropriate aria roles to align with the right aria pattern. [Menu Button](https://www.w3.org/WAI/ARIA/apg/patterns/menubutton/) + [Menu](https://www.w3.org/WAI/ARIA/apg/patterns/menu/)